src/main/java/frc/robot/Robot.java

```diff
@@ -56,12 +56,6 @@ public void disabledPeriodic() {}
   /** This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
   @Override
   public void autonomousInit() {
-    m_autonomousCommand = m_robotContainer.getAutonomousCommand();
```

Comment:
are we adding this stuff back? or is it not necessary for this year?

Reply:
I'm not sure, tbh. Idk if we'll stick to the default auton logic or not. Basel might have thought about it but we haven't talked about it.